cnn/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<head>
+<link rel="stylesheet" type="text/css" href="cnn.css">
+</head>
+<body>
+	<div class="containr">
+		<div class="ToolBar">
```

Comment:
Remember that in CSS we use lower-case instead of camelCase, e.g. tool-bar, hours-ago, etc.
See how clear is lowercase but Icon is uppercase? You will make some mistakes this way